__tests__/app.test.js

```diff
@@ -81,6 +82,52 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles (queries)", () => {
+  it("returns 200 and articles with a topic of 'mitch'", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.map((article) => {
+          expect(article).toHaveProperty("topic");
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+  it("returns 200 and sorts by any valid column", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("author", { descending: true });
+      });
+  });
+  it("returns 400 and rejects an invalid column", () => {
+    return request(app)
+      .get("/api/articles?sort_by=articles")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("invalid type");
+      });
+  });
+  it("returns results by created_at ASC", () => {
+    return request(app)
+      .get("/api/articles?order=ASC")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("created_at", { ascending: true });
+      });
+  });
+  it("returns 400 and rejects an invalid order", () => {
+    return request(app)
+      .get("/api/articles?order=order")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("invalid type");
+      });
+  });
+});
+
```

Comment:
A couple of tests to consider: 200 - valid topic but no articles / 404 - non-existent topic